Kickstarter-iOS/Views/Controllers/DiscoveryViewController.swift

```diff
@@ -3,12 +3,13 @@ import Library
 import Prelude
 import UIKit
 
-internal final class DiscoveryViewController: UIViewController {
+internal final class DiscoveryViewController: UIViewController, MessageBannerViewControllerPresenting {
```

Comment:
Wondering if we need to this for all the view controllers in the tab bar? Consider the situation where the user is in `SearchViewController` and then decides to go into their email and verify. They would be navigated back into the app but the banner won't show. Definitely a bit edge-casey.

Reply:
Going to look into this a bit more 👍